includes/framework/QType.class.php

```diff
@@ -68,13 +68,23 @@ public function __construct($strMessage, $intOffset = 2) {
 			throw new QCallerException('Type should never be instantiated.  All methods and variables are publically statically accessible.');
 		}
 
+		/** String Type */
 		const String = 'string';
+		/** Integer Type */
 		const Integer = 'integer';
+		/** Float Type */
 		const Float = 'double';
+		/** Boolean Type */
 		const Boolean = 'boolean';
+		/** Object Type */
 		const Object = 'object';
+		/** Array Type */
 		const ArrayType = 'array';
+		/** JSON - String Type but valid JSON Text */
+		const Json = 'json';
```

Comment:
I was thinking that we could use type casting for conversion. But I have pulled it back. It does not make much sense.